scripts/cangen/cangen/config.py

```diff
@@ -0,0 +1,47 @@
+import yaml
+
+
+class Bus:
+    def __init__(self, bus: dict):
+        self.bus = bus
+
+        self.dbc_file_path: str = self.bus.pop("dbcFile")
+        self.bus_name: str = self.bus.pop("busName").capitalize()
+
+        if not self.has_bus_been_consumed():
+            print(self.bus)
```

Comment:
You can pass a message to `raise ValueError(msg_goes_here)`. This message will be printed to the console when the error is raised (try it out). This is better practice than using `print`.

Replace the contents of the `if` block with

```python
raise ValueError(f"Unexpected bus keys {'.'.join(bus.keys())}")
```

Do the same thing for Config, but replace "bus" with "config"